investment-app/TRINV.Api/TRINV.Infrastructure/Interfaces/IApplicationDbContextFactory.cs

```diff
@@ -4,5 +4,4 @@
 
 public interface IApplicationDbContextFactory : IDbContextFactory<ApplicationDbContext>
```

Comment:
@glachezar Boom Good. Now let's think about the IApplicationDbContextFactory.

Our references: 

Application -> Domain
Infrastructure -> Application

Web -> Application and Infrastructure

Now if our interface for the DbContextFactory is inside the infrastructure layer. How we are gonna use it inside the application.  When the application layer doesn't know anything about the infrastructure. 
We will need it there right ?

Reply:
Easiest I would think is to make another abstraction in the application layer. 
But I will have to come back to you on that one.
Better to do some research on the matter as I am actually not sure if it is giong to work or it might be a better solution.

Reply:
DbContextFactory removed from application.